.travis.yml

```diff
@@ -1,11 +1,16 @@
-language: python
+language: c
```

Comment:
The multi-os feature doesn't work if the language is set to Python. BUT actually we don't use the built-in Python since we install it with conda hence why this will work.